webpack/ForemanTasks/Components/TasksTable/Components/SelectAllAlert.js

```diff
@@ -17,14 +17,13 @@ export const SelectAllAlert = ({
         Math.min(itemCount, perPage)
       )}
       <Button bsStyle="link" onClick={selectAllRows}>
-        {__('Select All')}
-        <b> {itemCount} </b> {__('tasks.')}
+        {sprintf(__('Select all %s tasks'), `<b>${itemCount}</b>`)}
       </Button>
     </React.Fragment>
   );
   const undoSelectText = (
     <React.Fragment>
-      {sprintf(__(`All %s tasks are selected. `), itemCount)}
+      {sprintf(__('All %s tasks are selected.'), itemCount)}
```

Comment:
This was translatable, but string interpolation is risky and it triggered my ``rg -l '__\(`'`` search. Also removes a trailing space from the translation. If JS needs the space, it should be done outside of the translation since a translator might do this for you and break the layout.